ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/LedgerTestContext.scala

```diff
@@ -11,9 +11,10 @@ import com.daml.ledger.api.testtool.infrastructure.Allocation.{
 import com.daml.ledger.api.testtool.infrastructure.participant.ParticipantTestContext
 
 import scala.concurrent.{ExecutionContext, Future}
+import scala.collection.immutable
 
 private[testtool] final class LedgerTestContext private[infrastructure] (
-    participants: Vector[ParticipantTestContext],
+    participants: immutable.Seq[ParticipantTestContext],
```

Comment:
What is the purpose of this? This is mainly making the performance characteristics unclear.

Reply:
This is generally tiny, on the order of 1–2 elements in the list. At this size, the performance characteristics are "meh, who cares"?